hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/SpanTypeAttributeEnricher.java

```diff
@@ -170,6 +171,13 @@ public static Protocol getGrpcProtocol(Event event) {
       }
     }
 
+    if (event.getRpc() != null && event.getRpc().getSystem() != null) {
```

Comment:
Should this be the first check, above the block above?

Reply:
Moved this.